commands/trigger.go

```diff
@@ -92,13 +93,24 @@ var triggerFireCmd = &cobra.Command{
 			return werr
 		}
 
+		if resp.StatusCode == http.StatusNoContent {
+			fmt.Fprintf(color.Output,
+				wski18n.T("trigger /{{.namespace}}/{{.name}} did not fire as it is not associated with an active rule\n",
```

Comment:
I suggest for all new message and to start transitioning these message ids to constants - see for example https://github.com/apache/incubator-openwhisk-cli/pull/315/commits/baa07f047767151c795a1caeaa6743c8370e8a9f.